### PR TITLE
Setup analysis

### PR DESCRIPTION
**Pull request by amelie-renard-sonarsource**

### CODE REVIEW

**Comment by arseniy-sonar** [comment]
@amelie-renard-sonarsource , can you, please review this PR and merge it if it is ready?

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  # Prepare the build system
+  - NUMBER_OF_PROCESSORS=$(nproc --all)
+  - mkdir build
+  - cmake -S . -B build
+  # Wraps the compilation with the Build Wrapper to generate configuration (used
+  # later by the SonarQube Scanner) into the "build_wrapper_output_folder" folder
+  - build-wrapper-macosx-x86 --out-dir "$BUILD_WRAPPER_OUT_DIR" cmake --build build/ --config Release
+  # And finally run the SonarQube analysis - read the "sonar-project.properties"
+  # file to see the specific configuration
+  - sonar-scanner -X --define sonar.cfamily.cache.enabled=true --define sonar.cfamily.cache.path=${TRAVIS_HOME}/.cfamily --define sonar.cfamily.threads=${NUMBER_OF_PROCESSORS} --define sonar.cfamily.build-wrapper-output="$BUILD_WRAPPER_OUT_DIR"
```
Comment (amelie-renard-sonarsource) [inline .travis.yml]:
```suggestion
  - sonar-scanner --define sonar.cfamily.cache.enabled=true --define sonar.cfamily.cache.path=${TRAVIS_HOME}/.cfamily --define sonar.cfamily.threads=${NUMBER_OF_PROCESSORS} --define sonar.cfamily.build-wrapper-output="$BUILD_WRAPPER_OUT_DIR"
```

File: README.adoc [hunk 2]
```diff
@@ -1,3 +1,42 @@
...
+
+*This project is analysed on https://sonarcloud.io/dashboard?id=sonarsource-cfamily-examples_macos-cmake-travis-sc[SonarCloud]!*
+
+:note-caption: :information_source:
+NOTE: This simple example ignores https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#analysis-cache[analysis cache] and https://docs.sonarcloud.io/advanced-setup/languages/c-c-objective-c/#parallel-code-scan[multithreaded execution] both of which can boost analysis significantly.
+
+It is very easy to analyze a C, C++ and Objective-C project with SonarCloud https://docs.travis-ci.com/user/sonarcloud/[on Travis CI]:
+
+. Create a `sonar-project.properties` file to store your configuration
+. Store the value of your token as SONAR_TOKEN environment variable in your Repository Settings
```
Comment (amelie-renard-sonarsource) [inline README.adoc]: Not true in this example, is it? The encrypted SONAR_TOKEN is written inside the `.travis.yml`

Comment (amelie-renard-sonarsource) [reply README.adoc]: The README of linux-autotools-travis-sc doesn't have this line whereas the README of linux-cmake-travis-sc has it.
I think we can remove it in both READMEs

Comment (arseniy-sonar) [reply README.adoc]: Indeed, well spotted

Comment (arseniy-sonar) [reply README.adoc]: removed.

**Comment by arseniy-sonar** [comment]
@amelie-renard-sonarsource , please take another look

**Pull request merged**